### ISSUE TITLE
Redis Sentinel

### ISSUE THREAD

**Issue by tonn2y** [StackExchange/StackExchange.Redis#455]
it's support Redis Sentinel?

**Comment by Ubloobok** [comment]
A long story exists in https://github.com/StackExchange/StackExchange.Redis/issues/22#issuecomment-226061137 , for now - without happy end.

**Comment by tonn2y** [comment]
thanks

**Comment by NickCraver** [comment]
The blunt version here is: we don't use Redis Sentinel, and are waiting on a critical cross-database replication feature for clustering as well. 

Anything we're not using in production doesn't get as much love because we can't dogfood it. We very much welcome community help on things that fall into this bucket, because we're at a fundamental disadvantage of providing the best solutions there. All of the other parts we're able to dogfood and heavily test in production at Stack Overflow.

**Comment by NickCraver** [comment]
Closing as a duplicate of #22 and #692
